docs/tutorial/getting-started.md

```diff
@@ -0,0 +1,147 @@
+# Tutorial: Deploy the Chrony charm
+
+## What you'll do
+This tutorial will walk you through deploying the chrony charm; you will:
+1. Deploy and configure the Chrony charm
+2. Enable NTS for the Chrony charm
+3. Use the Chrony charm as a time server
+
+## Prerequisites
+* LXD installed and bootstrapped.
+* A host machine with Juju version 3.4 or above and a Juju controller bootstrapped.
+
+## Create a Juju model
+Since the clock is a kernel function, we will need to use LXD virtual machines
+instead of containers.
+```
+juju add-model test-chrony
+juju set-model-constraints virt-type=virtual-machine
+```
+
+## Deploy the chrony charm and confirm the chrony charm are deployed
+1. Deploy the latest chrony charm.
```

Comment:
Nitpick: Remove the explicit numbering of steps (reads more like a how-to guide with numbered steps)

Reply:
Removed, thanks!